elastic/integrations
# Updating cluster overview dashboard for 24h time window

- Bug

## Proposed commit message


- WHAT: Updating `Memory used vs total memory` and `Cores used vs total cores` visualisations in Cluster Overview Dashboard
- WHY:  Those visualisations where failing when user was using 24h time window


## Checklist



## How to test this PR locally

1. Clone this repo
2. cd integrations/packages/kubernetes
3. run `elastic-package build`
4. Run `elastic-package stack up -d -v --version=8.14.0-SNAPSHOT`
5. Install kubernetes integration assets and verify changes

## Related issues

- Relates #https://github.com/elastic/kibana/issues/178623

## Screenshots

![Screenshot 2024-04-23 at 5 50 16 PM](https://github.com/elastic/integrations/assets/1685415/09bed9c4-2871-4b12-a179-b106c81dbf4e)

![Screenshot 2024-04-23 at 5 50 43 PM](https://github.com/elastic/integrations/assets/1685415/24d16dee-3496-40b3-8109-969ccbf62f27)

Changes applied:
![Screenshot 2024-04-23 at 5 50 55 PM](https://github.com/elastic/integrations/assets/1685415/774197c1-614f-4ba4-bbc2-7e582de3d161)
